charts/restate-helm/templates/service.yaml

```diff
@@ -10,7 +10,33 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
-  type: {{ .Values.service.type }}
+  ports:
+    - port: 9070
+      name: admin
+    - port: 8080
+      name: ingress
+    - port: 5122
+      name: node
+  clusterIP: None
```

Comment:
I believe this change will make the helm chart non upgradable, as you cannot update a service to remove its cluster ip. It also will remove a useful thing (a load balanced ip for all ready restate pods)

I suggest we make a new headless service, only for the node port, for use in discovery